spec/spec_helper.rb

```diff
@@ -33,12 +38,24 @@ def initialize(id = 'content_type')
 end
 
 class EntryDouble < Contentful::DynamicEntry
-  attr_reader :id, :content_type, :fields
+  attr_reader :id, :content_type
 
-  def initialize(id = '', content_type = ContentTypeDouble.new, fields = {})
+  def initialize(id = '', content_type = ContentTypeDouble.new, fields = {}, locales = false)
     @id = id
     @content_type = content_type
-    @fields = fields
+    @fields = {}
+    if locales
+      @fields = fields
+    else
+      fields.each do |k, v|
+        @fields['en-US'] ||= {}
```

Comment:
This can probably done outside of the `.each` loop, since it is not dependant on k,v, right?